.devcontainer/devcontainer.json

```diff
@@ -1,26 +1,45 @@
 {
     "name": "Pulumi",
-
     "build": {
         "dockerfile": "Dockerfile",
         "args": {
             "USER_NAME": "user",
             "USER_UID": "1000"
         }
     },
-
     "containerEnv": {
         "PULUMI_ACCESS_TOKEN": "${localEnv:PULUMI_ACCESS_TOKEN}",
         "PULUMI_TEST_ORG": "${localEnv:PULUMI_TEST_ORG}"
     },
-
     "remoteUser": "user",
-
-    "extensions": ["golang.go", "ms-dotnettools.csharp", "ms-python.python", "formulahendry.dotnet-test-explorer"],
-
     "postCreateCommand": "make ensure",
-
-    "settings": {
-        "extensions.ignoreRecommendations": true
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "golang.go",
+                "ms-dotnettools.csharp",
+                "ms-python.python",
+                "formulahendry.dotnet-test-explorer"
+            ],
+            "settings": {
+                "extensions.ignoreRecommendations": true,
+                "omnisharp.sdkPath": "/usr/local/dotnet/current"
```

Comment:
I would have thought it wouldn't need this, just picking up a standard install location by default?

Reply:
Yeah, I would think the same. I was pretty confused why the extension was erroring. I added the setting and it seemed to help the first time.